examples/skypilot/README.md

```diff
@@ -0,0 +1,158 @@
+# Running AReaL with SkyPilot
+
+This README includes examples and guidelines to running AReaL experiments with SkyPilot.
+Make sure you have SkyPilot properly installed following
+[our installation guide](../../docs/tutorial/installation.md#optional-install-skypilot)
+before running this example. Note that all command lines shown in this file are assumed
+to be execute under the root of AReaL repository.
+
+## Running a Single Node Experiment
+
+To run a single node experiment, you only need to setup the node with SkyPilot and
+launch the experiment with AReaL local launcher. [The following file](local.yaml) shows
+a SkyPilot yaml that could launch a simple GSM8K GRPO experiment in a single command
+line. This example runs on GCP, but could be easily migrated to other cloud or K8S
+cluster by changing `resource.infra` field in SkyPilot YAML file.
+
+```yaml
+name: areal-test-skypilot
+
+resources:
+  infra: gcp
+  accelerators: A100:2
+  autostop:
+    idle_minutes: 10
+    down: true
+  cpus: 8+
+  memory: 32GB+
+  disk_size: 256GB
+  image_id: docker:ghcr.io/inclusionai/areal-runtime:v0.3.4
+
+
+num_nodes: 1
+
+workdir: .
+
+envs:
+  EXPERIMENT_NAME: my-areal-experiment
+  TRIAL_NAME: my-trial-name
+
+run: |
+  python3 -m areal.launcher.local examples/math/gsm8k_grpo.py \
+    --config examples/math/gsm8k_grpo.yaml \
+    experiment_name=$EXPERIMENT_NAME \
+    trial_name=$TRIAL_NAME \
+    cluster.n_gpus_per_node=2 \
+    allocation_mode=sglang.d1+d1 \
+    train_dataset.batch_size=4 \
+    actor.mb_spec.max_tokens_per_mb=4096
+```
+
+To run the experiment, execute:
+
+```bash
+sky launch -c areal-test examples/skypilot/local.yaml
+```
+
+## Running a Multi-Node Experiment
+
+### Running AReaL with Ray Launcher
+
+The following example shows how to setup a ray cluster with SkyPilot and then use AReaL
+to run GRPO with GSM8K dataset on 2 nodes, each with 1 A100 GPU. This example runs on
```

Comment:
What's the point of using "2 nodes, each with 1 A100 GPU" instead of a single node with several GPUs? 
This training will be much slower due to the slow interconnectivity speed between nodes.